lib/logger.js

```diff
@@ -11,8 +11,44 @@ Hoek.assert(process.env.LOG_LEVEL, 'Must define LOG_LEVEL');
 var stream = new PrettyStream();
 stream.pipe(process.stdout);
 
+function requestSerializer(req) {
+  return {
+    path: req.path,
+    method: req.method,
+    headers: req.headers
+  };
+}
+
+function responseSerializer(res) {
+  return {
+    payload: res.output.payload
+  };
+}
+
+function errorSerializer(res) {
+  if (!res.data) {
```

Comment:
I'm torn on this one. 

On the one hand, the entire point of the serializer is to parse this sort of stuff for us, allowing the user to log objects without worrying about what they contain. On the other hand, we have a dedicated function for doing the *only log in the entire request*, so it isn't unreasonable to do it there. 

I'm going to say these checks should stay here.